src/main/java/commands/SearchCommand.java

```diff
@@ -87,16 +124,36 @@ public void searchExercise() {
         }
     }
 
-    public boolean isMatchExercise(String exercise) {
+    public boolean isMatch(String exercise) {
         return exercise.contains(userArguments);
     }
 
+    public void searchPlan() throws InvalidCommandException {
+        var planListToSearch = planList.getPlansDisplayList();
+        for (int i = 0; i < planListToSearch.size(); i++) {
+            String planToBeCompared = planListToSearch.get(i).toLowerCase();
+            var planHash = planList.getPlanFromKey(planToBeCompared);
+            var workouts = planHash.getWorkoutsInPlanList();
```

Comment:
Just wondering, are these variables used anywhere?

Reply:
I forget to remove, thank you!